src/extract-metadata.js

```diff
@@ -157,35 +246,36 @@ async function extractMetaData(context, action) {
   const { document } = content;
   const { url } = request;
 
-  // extract metadata
   const { meta = {} } = content;
   content.meta = meta;
 
-  const metaBlock = document.querySelector('body div.metadata');
-  if (metaBlock) {
-    const metaConfig = readBlockConfig(metaBlock);
-    [
-      // supported metadata properties
-      'title',
-      'description',
-      'keywords',
-      'tags',
-      'image',
-    ].forEach((name) => {
-      if (metaConfig[name]) {
-        meta[name] = metaConfig[name];
-        delete metaConfig[name];
-      }
-    });
-    if (Object.keys(metaConfig).length > 0) {
-      // add rest to meta.custom
-      meta.custom = Object.keys(metaConfig).map((name) => ({
-        name,
-        value: metaConfig[name],
-        property: name.includes(':'),
-      }));
+  // extract global metadata from spreadsheet, and overlay
+  // with local metadata from document
+  const metaConfig = Object.assign(
+    await getGlobalMetadata(getPath(request), action),
```

Comment:
in case `getGlobalMetadata` throws an error... do you want to abort the entire request, or just continue with no external metadata ?

Reply:
The latter... I'll add try/catch 😉